docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -135,25 +138,61 @@ def get(self, request, projectid, filename):
 
     # TODO refactor this function by moving the actual upload and Project model updates to library function outside the view
     def post(self, request, projectid, filename, format=None):
+        # Send traceback to buffer for debugging purposes
+        buffer = io.StringIO()
+        print_stack(limit=20, file=buffer)
+
         project = Project.objects.get(id=projectid)
 
+        # Send Sentry the ISO-8859-1-decoded request body
+        # as well as the traceback to help investigate cases where
+        # 'file' is missing from request.data
+        try:
+            with configure_scope() as scope:
+                name = f"{request.META.get('X-Request-Id')}_{projectid}_{filename}.txt"
+                prefix = "request_body_"
+                scope.add_attachment(
+                    bytes=bytes(request.body.decode("ISO-8859-1"), encoding="utf-8"),
+                    filename=prefix + name,
+                )
+                prefix = "traceback_"
+                scope.add_attachment(
+                    bytes=bytes(buffer.getvalue(), encoding="utf-8"),
+                    filename=prefix + name,
+                )
+        except Exception as error:
+            logging.info(f"Unable to send file to Sentry: failed on {error}")
```

Comment:
If you confirmed this is working, I would move it within `if "file" not in request.data:` condition a few lines below, otherwise we will run out of storage on Sentry very quickly.